src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.1);
+
+}
+
+.logo__link {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__links {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav___link {
+  display: inline-block;
+  margin: 0 10px;
+  line-height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  cursor: pointer;
+  color: black;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.first-item,
+.is-active {
+  margin-left: 0;
+}
+
+.last-item,
+.last-link {
+  margin-right: 0;
+}
```

Comment:
[Tree-structural pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes#tree-structural_pseudo-classes)

Reply:
this is ins't fixed, use pseudo-classes